### PR TITLE
Projects use media for images and og tags display width and height

### PR DESCRIPTION
**Pull request by robbiecarlton**
This should not be merged until https://github.com/Hylozoic/hylo-node/pull/57 has been merged, and the database updated

### CODE REVIEW

**Comment by robbiecarlton** [comment]
@levity lm if this is good to merge once the other branch and db updates are in

**Comment by levity** [comment]
i'll look at this once the other one's merged so that the diff is clearer

**Comment by robbiecarlton** [comment]
:+1:

**Comment by robbiecarlton** [comment]
@levity :shipit:?

File: api/services/GetImageSize.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+var Promise = require('bluebird')
+var url = require('url')
+var http = require('http')
+var https = require('https')
+
+module.exports = function (image_url) {
+  return new Promise(function (resolve, reject) {
+    var options = url.parse(image_url)
+    var protocol = (options.protocol === 'https:' ? https : http)
+    protocol.get(options, function (response) {
```
Comment (levity) [inline api/services/GetImageSize.js]: how about using the `request` module for this?

File: api/services/GetImageSize.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+var imageSize = require('image-size')
+var Promise = require('bluebird')
+var url = require('url')
+var http = require('http')
+var https = require('https')
+
+module.exports = function (image_url) {
```
Comment (levity) [inline api/services/GetImageSize.js]: camelCase please :camel:

**Comment by levity** [comment]
looks :ship:-able otherwise

**Pull request merged**